tests/cli/typegen.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { spawnSync, SpawnSyncReturns } from "child_process";
```

Comment:
Had to resort to integration tests because oclif is tightly coupled to mocha https://oclif.io/docs/testing